### PR TITLE
Changelog for v0.11.0

### PR DESCRIPTION
**Pull request by qiujian16**

### CODE REVIEW

**Comment by skeeey** [comment]
LGTM

File: CHANGELOG/CHANGELOG-v0.11.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# v0.11.0
+
+## Changelog since v0.11.0
+
+### New Features
+* Add installStrategy in addon v1alpha1 API ([#193](https://github.com/open-cluster-management-io/api/pull/193) [@qiujian16](https://github.com/qiujian16))
```
Comment (haoqing0110) [inline CHANGELOG/CHANGELOG-v0.11.md]: should this point to https://github.com/open-cluster-management-io/api/pull/207 ?

Comment (qiujian16) [reply CHANGELOG/CHANGELOG-v0.11.md]: good catch

**Pull request merged**